src/components/Chart/AreaChart.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import C3Chart from 'react-c3js';
+import ClassNames from 'classnames';
+
+require('patternfly/dist/js/patternfly-settings');
+require('patternfly/dist/js/patternfly-settings-charts');
+
+const c3ChartDefaults = window.patternfly.c3ChartDefaults();
+
+/**
+ * AreaChart Component for Patternfly React
+ */
+class AreaChart extends React.Component {
+  render() {
+    this.props.data.type = this.props.data.type || 'area';
```

Comment:
does this works? I was under the assumption you can't change props.

Reply:
you can not do `this.props.data = ... `
but you can do:
`this.props.data.type = ... `